### PR TITLE
Make default behavior for bootloader to enable LDOTRIM

### PR DESCRIPTION
**Pull request by cnlohr**
Yuuki Umeta has had issues on a few boards not working reliably but on those boards, when we enabled LDOTRIM, the boards worked perfectly. 

This is just changing that for the bootloader.  We can investigate other things later.

### CODE REVIEW

**Comment by YuukiUmeta-UIAP** [comment]
DON'T push it. I found out the another issue caused fixing LDOTRIM code.

First, I wrote "demo_hidapi" to new empty 100 boards without `EXTEND->CTR |= EXTEN_LDO_TRIM;`. Then, 2 boards had communication problems via "testtop". The remaining 98 boards worked fine.

Second, I wrote "demo_hidapi" to identical 100 boards with `EXTEND->CTR |= EXTEN_LDO_TRIM;`. Then 4 boards that aren't the first 2 bad had communication problems via "testtop". The remaining 96 boards worked fine.

In other words, depending on the `EXTEND->CTR |= EXTEN_LDO_TRIM;`, 100 boards can be classified into the following three patterns:

- 94 boards work fine with or without the fixing code.
- 2 boards work fine only with the fixing code.
- 4 boards work fine only without the fixing code.

This issue is reproducible on my PC. 2 and 4 bad boards were tried to switch the fixing code many times because it was unbelievable for me.

**Comment by YuukiUmeta-UIAP** [comment]
> DON'T merge it. I found out the another issue caused fixing LDOTRIM code: `EXTEND->CTR |= EXTEN_LDO_TRIM;`
> 
> First, I wrote "demo_hidapi" to new empty 100 boards without the fixing code. Then, 2 boards had USB communication problems via "testtop". The remaining 98 boards worked fine.
> 
> Second, I wrote "demo_hidapi" to identical 100 boards with the fixing code. Then 4 boards that aren't the first 2 bad had USB communication problems via "testtop". The remaining 96 boards worked fine.
> 
> In other words, depending on the fixing code, 100 boards can be classified into the following three patterns:
> 
> * 94 boards work fine with or without the fixing code.
> * 2 boards work fine only with the fixing code.
> * 4 boards work fine only without the fixing code.
> 
> This issue is reproducible on my PC. 2 and 4 bad boards were tried to switch the fixing code many times because it was unbelievable for me.

The quantity, 2 or 4, is not important. In other words, it does not mean that I recommend you to adopt the code that will cause fewer issues. Because the number of surveyed cases is small, it is not correct to take this number as a probability.

**Comment by cnlohr** [comment]
*holding off on merge*

**Comment by YuukiUmeta-UIAP** [comment]
I have just understood it fixed with #57. Thank you!
